zebrad/src/components/mempool/crawler/tests/timing.rs

```diff
@@ -0,0 +1,29 @@
+//! Timing tests for the mempool crawler.
+
+use std::convert::TryInto;
+
+use zebra_chain::parameters::POST_BLOSSOM_POW_TARGET_SPACING;
+use zebra_network::constants::{DEFAULT_CRAWL_NEW_PEER_INTERVAL, HANDSHAKE_TIMEOUT};
+
+use crate::components::mempool::crawler::RATE_LIMIT_DELAY;
+
+#[test]
+fn ensure_timing_consistent() {
```

Comment:
Nit: adding a verb to the name for extra clarity:

```suggestion
fn ensure_timings_are_consistent() {
```